.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,26 +10,23 @@
   - Each commit in the pull request has a meaningful commit message
 
   - Once all items of the checklist are addressed, remove the above text and this checklist, leaving only the filled out template below.
-
-**(The sections below can be removed for hotfixes of typos)**
```

Comment:
This is not true. We should retain the documentation section since it's read by actions. Also, I tend not to open the door for "hotfixes". If the change is trivial, taking some time to go through the template doesn't hurt, as long as the template is concise.